jobbergate-agent-snap/Makefile

```diff
@@ -20,6 +20,10 @@ build: ## Build the snap
 	@git rev-parse --short HEAD > sha/commit-sha.txt
 	@snapcraft $(ARGS)
 
+.PHONY: changes
+changes:
+	towncrier create --dir .. --section Agent
```

Comment:
I'd say the snap could have its own section instead of using the Agent's

Reply:
I agree.

Reply:
Added: [9759b96](https://github.com/omnivector-solutions/jobbergate/pull/690/commits/9759b96ae74e5169d0eb86da06ca22df44d3ac40).